packages/endpoints-organizations-registrar/src/entities/organization-services/orchestrators/init-add-service-to-organization.js

```diff
@@ -57,9 +57,13 @@ const initAddServiceToOrganization = (fastify) => {
     }
 
     // add consent
-    await repos.registrarConsents.insert(
-      buildServiceConsent(organization, createdService, context)
-    );
+    await repos.registrarConsents.registerConsent({
+      userId: user.sub,
+      organization,
```

Comment:
The rest of the code takes in scalar values, and we CAN be consistent here so this should be `organizationId: organization?._id` or else retain the `build` fn